src/apps/vpn/dnshelper.cpp

```diff
@@ -49,22 +57,52 @@ QString DNSHelper::getDNS(const QString& fallback) {
   }
 
   static QMap<int, QString> dnsMap{
```

Comment:
Instead of doing all of this, change the map to be:

QMap<int, DNSData>. Where DNSData is a simple struct composed by IP address and telemetry name.

Then use the telemetry name for the glean event.

What do you think?

Reply:
I like that suggestion *a lot*. I just implemented this, let me know if you were thinking about something different.